apps/asap-server/src/utils/migrations.ts

```diff
@@ -31,3 +32,19 @@ export const applyToAllItemsInCollection = async <
     pointer += 10;
   } while (pointer < result.total);
 };
+
+export const loopOverCustomCollection = async <TResponse>(
```

Comment:
i think this method should not really belong to `migrations`

Reply:
It has to be somewhere.
I find this place as it has some loop-over collection.
Where do you want to place it?

Reply:
migration should imho be part of squidex package except then we wouldn't be able to use the controllers (maybe we shouldn't, dont know). Can you move it to `utils`?

Reply:
but it is in utils :D